src/commands/shared/request.ts

```diff
@@ -108,13 +108,19 @@ export const request =
           return "Requesting access";
       }
     };
-    const data = await spinUntil(
-      accessMessage(options?.message),
-      fetchCommand<RequestResponse<T>>(resolvedAuthn, args, [
-        command,
-        ...args.arguments,
-      ])
-    );
+    const data =
+      options?.message != "quiet"
+        ? await spinUntil(
+            accessMessage(options?.message),
+            fetchCommand<RequestResponse<T>>(resolvedAuthn, args, [
```

Comment:
DRY: it looks like the command we execute is the same so we can encapsulate the function as a variable and call it in the two places we need.